<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvWindow</class>
 <widget class="QMainWindow" name="AdvWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(218, 226, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>20</y>
      <width>401</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>83</x>
       <y>13</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #12222D;
font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 120%;
text-align: center;</string>
     </property>
     <property name="text">
      <string>ПРОДВИНУТЫЕ НАСТРОЙКИ</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>30</y>
      <width>94</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/back.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>771</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/advField.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/defaultAdv.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="saveLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/saveAdv.png</pixmap>
    </property>
   </widget>
   <widget class="QSlider" name="speedVatSlider">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>117</y>
      <width>271</width>
      <height>38</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="fullVatSlider">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>168</y>
      <width>271</width>
      <height>38</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="speedDoorSlider">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>271</width>
      <height>38</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 14px;
line-height: 130%;
color: #12222D;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Скорость поворота ванны</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 14px;
line-height: 130%;
color: #12222D;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Наполняемость ванны</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 14px;
line-height: 130%;
color: #12222D;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Скорость открытия двери</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>119</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 24px;
color: #12222D;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>168</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 24px;
color: #12222D;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>214</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 24px;
color: #12222D;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 14px;
line-height: 130%;
color: #12222D;
border:0px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'RF Dewi'; font-size:14px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14px; font-weight:696;&quot;&gt;Дистанция подъёма платформы&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 14px;
line-height: 130%;
color: #12222D;
border:0px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'RF Dewi'; font-size:14px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14px; font-weight:696;&quot;&gt;Задержка начала поворота ванны&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 14px;
line-height: 130%;
color: #12222D;
border:0px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'RF Dewi'; font-size:14px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14px; font-weight:696;&quot;&gt;Задержка окончания поворота ванны&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/change.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/change.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/change.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>411</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 14px;
color: #12222D;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>мм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>411</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 14px;
color: #12222D;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>сек</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>411</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 14px;
color: #12222D;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>сек</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upPlusButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upMinusButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startVatMinusButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startVatPlusButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="finishVatMinusButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="finishVatPlusButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="defaultButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="upText">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>370</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 900;
font-size: 26px;
color: #12222D;
background-color: rgba(255, 255, 255, 0);
border:0px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>1.2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="startVatText">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>370</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 900;
font-size: 26px;
color: #12222D;
background-color: rgba(255, 255, 255, 0);
border:0px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>1.2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="finishVatText">
    <property name="geometry">
     <rect>
      <x>397</x>
      <y>370</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 900;
font-size: 26px;
color: #12222D;
background-color: rgba(255, 255, 255, 0);
border:0px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>1.2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="speedVatText">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>113</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 900;
font-size: 26px;
color: #12222D;
background-color: rgba(255, 255, 255, 0);
border:0px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fullVatText">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>163</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 900;
font-size: 26px;
color: #12222D;
background-color: rgba(255, 255, 255, 0);
border:0px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="speedDoorText">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>209</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 900;
font-size: 26px;
color: #12222D;
background-color: rgba(255, 255, 255, 0);
border:0px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>75</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="connectionWifi">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/noSignal.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
